R/gwlAreaPlot.R

```diff
@@ -119,21 +114,40 @@ gwl_area_plot <- function(dataframe, trend, intercept, state, sig,
                  expand = c(0,0)) + 
     scale_fill_manual(name = '', values = c('Groundwater Level' = "#1E90FF"))
   
-  vals <- c(LTT = 'orange', Interp = 'grey30')
+  vals <- c(LTT = 'orange', Interp = 'grey60')
   labs <- c('Long-term Trend', 'Interpolated (Missing) Values')
-  override_list <- list(colour = c("orange", "grey30"), shape = c(NA, 16), linetype = c(1, 0))
+  override_list <- list(colour = c("orange", "grey60"), shape = c(NA, 16), linetype = c(1, 0))
   
   if (showInterpolated) {
     plot.area <- plot.area + 
       geom_point(data = df[df$nReadings == 0,],
                  aes_string(y = "med_GWL", colour = "'Interp'"),
-                 size = 1)
+                 size = 0.5)
   } else {
     vals <- vals[1]
     labs <- labs[1]
     override_list <- lapply(override_list, `[`, 1)
   }
   
+  if (show_stable_line) {
```

Comment:
This is great. I think it could be simplified by lumping the logic a bit - does this make sense?

```r
if (show_stable_line || tolower(trend_category) != "stable" {
  plot.area <- plot.area + 
    geom_abline(aes_string(intercept = "intercept", slope = "slope", colour = "'LTT'"),
                data = data.frame(intercept = -int.well, slope = slope), size = 1)
} else if (tolower(trend_category) == "stable") {
  vals <- vals[2]
  labs <- labs[2]
  override_list <- lapply(override_list, `[`, 2)
}
```

Reply:
Yes, I was sure there was a more efficient path than an embedded `if/else`. I had not seen the  `||` or the `if/else if`. Cool.